src/sqlfluff/core/rules/base.py

```diff
@@ -456,7 +456,7 @@ def _validate_config_options(self, config, rule=None):
         Config options can also be checked for a specific rule e.g L010.
         """
         rule_config = config.get_section("rules")
-        from icecream import ic
+        #from icecream import ic

```

Comment:
I think this was debugging code? I didn't find an `icecream` package in the requirements, so I commented it out.